tests/static/test_strategies/download.py

```diff
@@ -34,10 +34,14 @@ def initialize(
         self, session: "Optional[Dict[str, Any]]" = None
     ) -> "Dict[str, Any]":
         """Initialize"""
+        del session
+        dummy = self.resource_config.deepcopy()
+        del dummy
```

Comment:
What's the motivation behind deleting the incoming variable?
What's more, why delete the created class attribute? It should/will be deleted as soon as the immediate request is resolved anyway.

Reply:
Never mind - I see now that these are all for the test strategies used for testing, i.e., deleting the variables here makes sense for the purpose of test-isolation 👍